front-end/pom.xml

```diff
@@ -184,14 +183,14 @@
                         <goal>exec</goal>
                     </goals>
                     <configuration>
-                        <executable>${basedir}/target/node/bin/jsx</executable>
+                        <executable>node</executable>
                         <environmentVariables>
                             <PATH>${env.PATH}:${pom.basedir}/target/node/node</PATH>
```

Comment:
This `PATH` setting doesn't help to find the executable itself, it's only passed to the child process AFAICT.